Backend/purrfect-match-backend/src/main/java/com/restservice/controller/RegistrationController.java

```diff
@@ -0,0 +1,40 @@
+package com.restservice.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailException;
+import org.springframework.web.bind.annotation.RestController;
+
+import com.restservice.model.ContactPage;
+import com.restservice.service.MailService;
+
+import org.springframework.web.bind.annotation.CrossOrigin;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+
+
+ //Mail API using Spring Boot
+@RestController
+public class RegistrationController {
```

Comment:
Can you think of a better name than RegistrationController? What's getting registered?